### ISSUE TITLE
App Crash : in Lolipop

### ISSUE THREAD

**Issue by GauravCreed** [hotchemi/Android-Rate#98]
Hi Shintaro Katafuchi

you have done a superb work on displaying dialog, it perfectly work on lower version but when i am installing in my Phone (Asus Device : Android Lolipop v : 5.0.1)  it will crash app so please help me to sort out this issue as possible as fast

the Error Log like this.

```
03-26 16:14:35.133: E/libEGL(26083): eglMakeCurrent start
03-26 16:14:35.228: E/libEGL(26083): eglMakeCurrent start
03-26 16:14:35.249: E/AndroidRuntime(26083): FATAL EXCEPTION: main
03-26 16:14:35.249: E/AndroidRuntime(26083): Process: com.appshub.contactsbackup.free, PID: 26083
03-26 16:14:35.249: E/AndroidRuntime(26083): java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.internal.app.WindowDecorActionBar.getDecorToolbar(WindowDecorActionBar.java:248)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.internal.app.WindowDecorActionBar.init(WindowDecorActionBar.java:201)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.internal.app.WindowDecorActionBar.<init>(WindowDecorActionBar.java:184)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.app.AppCompatDelegateImplV7.createSupportActionBar(AppCompatDelegateImplV7.java:176)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.app.AppCompatDelegateImplBase.getSupportActionBar(AppCompatDelegateImplBase.java:85)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.app.AppCompatDelegateImplV7.onStop(AppCompatDelegateImplV7.java:221)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.app.AppCompatDialog.onStop(AppCompatDialog.java:108)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.app.Dialog.dismissDialog(Dialog.java:361)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.app.Dialog.dismiss(Dialog.java:337)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:157)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.os.Handler.dispatchMessage(Handler.java:111)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.os.Looper.loop(Looper.java:194)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at android.app.ActivityThread.main(ActivityThread.java:5631)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at java.lang.reflect.Method.invoke(Native Method)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at java.lang.reflect.Method.invoke(Method.java:372)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:959)
03-26 16:14:35.249: E/AndroidRuntime(26083): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:754)
```

thanks.

**Comment by hotchemi** [comment]
I'll fix it soon and please also check #99 for the time being.

**Comment by hotchemi** [comment]
Could you check with 1.0.1?
